packages/formation-react/src/components/modal/Modal.jsx

```diff
@@ -29,9 +30,9 @@ class Modal extends React.Component {
   }
 
   setupModal() {
-    this.applyFocusToModal();
+    this.applyFocusToFirstModalElement();
```

Comment:
I renamed this existing method since we will now be programmatically focusing the last element in the modal as well as the first.